### PR TITLE
setup-mobile-completion

### PR DESCRIPTION
**Pull request by ClarenceAtim**
modified html and css,
 added images

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,7 +42,156 @@ <h1>Hey There.<br>I'm Clarence</h1>
...
+        <p>
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s,
+            when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="link-1 btn">
+            <li><a href="#">CSS</a></li>
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">RUBY</a></li>
```
Comment (wayungi) [inline index.html:61]: - [ ] Please note the color differences between your buttons & the figma design & Kindly have it fixed below.

**figma**
btns1

**Your design**
btns2

File: style.css [hunk 2]
```diff
@@ -77,3 +77,249 @@ header a {
...
+  border-bottom: 1px solid #212121;
+  font-size: 40px;
+  line-height: 52px;
+  font-style: normal;
+  font-weight: 400;
+  width: 327px;
+  height: 83px;
+}
+
+.card-2 {
```
Comment (wayungi) [inline style.css:101]: - [ OPTIONAL] The class `.card-2` & `card-3` have a lot in common. You can write **DRY** CSS here by combing the common properties like those below 👀 
```
.card-2, .card-3 {
   display: flex;
   flex-direction: column;
   ...
 } 
```

File: index.html [hunk 3]
```diff
@@ -42,7 +42,156 @@ <h1>Hey There.<br>I'm Clarence</h1>
...
+  </section>
+  <section class="about-myself">
+    <h2>About me</h2>
+    <p>
+        Hello I'm a software developer! I can help you build a product ,
+        feature or website Look through some of my work and experience!
+        If you like what you see and have a project you need coded, don't hesitate to contact me.
+    </p>
+    <a class="see-project button-1" href="#">Get my resume</a>
+    <img src="images/Frame.svg" alt="frame image">
```
Comment (wayungi) [inline index.html]: - [ ] Kindly note that this is supposed to be a background image. Kindly add it using CSS with the *background-image* property. 👍🏾

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @ClarenceAtim , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] All linter checks are passing ✔️ 
- [ ]  Correct GitHub flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @ClarenceAtim,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

- [OPTIONAL] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**